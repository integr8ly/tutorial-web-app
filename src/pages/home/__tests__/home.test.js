import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../home';

describe('HomePage Component', () => {
  it('should render the home page component', () => {
    const component = shallow(<HomePage />);

    expect(component).toMatchSnapshot();
  });
});
