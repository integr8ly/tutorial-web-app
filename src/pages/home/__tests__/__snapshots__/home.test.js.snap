// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Component should render the home page component 1`] = `
<Grid
  bsClass="container"
  componentClass="div"
  fluid={true}
>
  <div
    className="page-header"
  >
    <h2>
      Welcome to PatternFly React Product Demo App!
    </h2>
  </div>
  <div
    className="app-body"
  >
    <Alert
      className=""
      onDismiss={[Function]}
      type="success"
    >
      <span>
        Well done! You've installed this demo correctly.
      </span>
    </Alert>
    <div
      className="app-intro"
    >
      <img
        alt="logo"
        className="app-logo"
        src="logo.svg"
      />
      <h2>
        Welcome to PatternFly React
      </h2>
    </div>
    <p
      className="app-paragraph"
    >
      To get started, edit 
      <code>
        src/app.js
      </code>
       and save to reload.
    </p>
  </div>
</Grid>
`;
